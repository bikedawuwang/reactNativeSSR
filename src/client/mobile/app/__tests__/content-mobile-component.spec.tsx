import * as React from 'react';
import { shallow } from 'enzyme';

// Mobile
import SharedComponentMobile from '../../../shared/components/content';

describe('SharedComponent <SharedComponent />', () => {
	it('Shared Component Mobile renders correctly', () => {
		const sharedComponentMobile = shallow(<SharedComponentMobile />);

		// Interaction demo
		expect(sharedComponentMobile.text()).toEqual('<App />');

		// Snapshot demo
		expect(shallow).toMatchSnapshot();
	});
});
