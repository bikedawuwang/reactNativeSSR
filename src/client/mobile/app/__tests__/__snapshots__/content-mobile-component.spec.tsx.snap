// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SharedComponent <SharedComponent /> Shared Component Mobile renders correctly 1`] = `[Function]`;
